src/Fabulous/Program.fs

```diff
@@ -3,12 +3,19 @@ namespace Fabulous
 open System
 open System.Diagnostics
 
+(*TODO Is either of these a program in the Elm sense? If so, where's the view in this one?
+Or are these rather abstractions of or pre-cursors to an Elm Program?
+AFAIU in the Elm architecture a "program" manages the application's (or component's) state, actions, and view rendering.
+Please help me as a MVU/Elm newbie understand these types. *)
+//TODO what's the 'arg?
```

Comment:
`'arg` is a value you can pass to initialize a program. This `'arg` will be received by the `init` function.

```fs
module App =
    let init (isAdmin: bool) =
        { Model.Name = if isAdmin then "Admin" else "User" }
    
    let program = 
        Program.stateful init update
        |> Program.withView view

type MauiProgram =
    static member CreateMauiApp() =
        let isAdmin = true

        MauiApp
            .CreateBuilder()
            .UseFabulousApp(App.program, isAdmin)
```